### ISSUE TITLE
fbx not supported

### ISSUE THREAD

**Issue by cissusnar** [dmsurti/AssimpKit#84]
when I load a fbx file, it will print following errors:

2017-11-02 14:50:19.971128+0800 OSX-Example[12918:245460] errors encountered while discovering extensions: Error Domain=PlugInKit Code=13 "query cancelled" UserInfo={NSLocalizedDescription=query cancelled}

**Comment by dmsurti** [comment]
FBX is supported. Please attach the fbx file or a sample fbx file that demonstrates this error for further investigation. Thanks.

**Comment by cissusnar** [comment]
sorry, just forgot upload sample.  :P

[Sapphiart_model.fbx.zip](https://github.com/dmsurti/AssimpKit/files/1436798/Sapphiart_model.fbx.zip)

**Comment by dmsurti** [comment]
No worries. I just checked and it is a problem with the `assimp` library that `AssimpKit` uses to import the models.

The error reported is: 

```
Scene importing failed with error FBX-Tokenize (offset 0x34) block offset is out of range
```

I have reported this [error upstream to `assimp`](https://github.com/assimp/assimp/issues/1527).  Once it is fixed there, AssimpKit will handle it fine. Closing this for now as it is dependent on a fix upstream.
